articles/virtual-machines/windows/quick-create-powershell.md

```diff
@@ -52,35 +52,16 @@ New-AzVm `
     -OpenPorts 80,3389
 ```
 
-## Connect to virtual machine
-
-After the deployment has completed, RDP to the VM. To see your VM in action, the IIS web server is then installed.
-
-To see the public IP address of the VM, use the [Get-AzPublicIpAddress](/powershell/module/az.network/get-azpublicipaddress) cmdlet:
-
-```azurepowershell-interactive
-Get-AzPublicIpAddress -ResourceGroupName 'myResourceGroup' | Select-Object -Property  'IpAddress'
-```
-
-Use the following command to create a remote desktop session from your local computer. Replace `publicIpAddress` with the public IP address of your VM. 
-
-```powershell
-mstsc /v:publicIpAddress
-```
-
-In the **Windows Security** window, select **More choices**, and then select **Use a different account**. Type the username as **localhost**\\*username*, enter password you created for the virtual machine, and then click **OK**.
-
-You may receive a certificate warning during the sign-in process. Click **Yes** or **Continue** to create the connection
-
 ## Install web server
 
 To see your VM in action, install the IIS web server. Open a PowerShell prompt on the VM and run the following command:
 
-```powershell
-Install-WindowsFeature -Name Web-Server -IncludeManagementTools
+```azurepowershell-interactive
+Invoke-AzVMRunCommand -ResourceGroupName 'myResourceGroup' -VMName 'myVM' -CommandId 'RunPowerShellScript' -ScriptString 
+'Install-WindowsFeature -Name Web-Server -IncludeManagementTools'
 ```
 
-When done, close the RDP connection to the VM.
+The `-ScriptString' parameter requires version `4.27.0` or later of the 'Az.Compute` module.

```

Comment:
```suggestion
The `-ScriptString` parameter requires version `4.27.0` or later of the `Az.Compute` module.

```
Replacing quote marks with back ticks to create correct rendering for inline code strings.